docs/core/testing/unit-testing-mstest-runner-extensions.md

```diff
@@ -8,15 +8,17 @@ ms.date: 12/15/2023
 
 # MSTest runner extensions
 
-The MSTest runner supports extensions that can be used to customize the behavior of the test runner.
+The MSTest runner supports extensions that can be used to customize the behavior of the test runner. These extensions are not part of the default bundle to offer biggest flexibility.
```

Comment:
```suggestion
The MSTest runner can be customized through extensions. These extension are either built-in or can be installed as NuGet packages. Extensions installed throught NuGet packages will auto-register the extensions they are holding to become available in test execution (unless you [disable it](#extension-registration)).
```

Not sure what you meant here  by "the default bundle", e.g. trx is imho already part of MSTest dependencies, because mstest -> mstest.adapter -> vstestbridge -> Microsoft.Testing.Platform.Extensions.TestReports.TrxReport 

Nevertheless I think it is valuable to document all the "major" extensions we made available, no matter if they ship together with mstest or not. Especially if they have their counterpart in vstest.

The extension registration entry might describe how extensions are registered, and what you need to do when you disable program.cs autoregistration. Each extension might also show example of the registration code? Or is that too much detail for now?

Reply:
> Not sure what you meant here by "the default bundle", e.g. trx is imho already part of MSTest dependencies, because mstest -> mstest.adapter -> vstestbridge -> Microsoft.Testing.Platform.Extensions.TestReports.TrxReport

this dep is the object model, so actually as-is is not enabling the trx support

Reply:
Can it be renamed Microsoft.Testing.Platform.Extensions.TestReports.TrxReport**Primitives**  or Abstractions? or something like that, because the current name is strongly suggesting (to me) that this is the actual extension (there is extension in the name, and no *abstract* name)

Reply:
```suggestion
The MSTest runner can be customized through extensions. These extension are either built-in or can be installed as NuGet packages. Extensions installed throught NuGet packages will auto-register the extensions they are holding to become available in test execution.
```